lib/ansible/utils/plugin_docs.py

```diff
@@ -179,17 +186,18 @@ def add_fragments(doc, filename, fragment_loader, is_module=False):
                     doc['seealso'] = []
                 doc['seealso'].extend(seealso)
 
-        if 'options' not in fragment:
```

Comment:
The `add_collection_to_versions_and_dates` call in line 173 needs `return_docs=True` passed if this is a return value fragment.